src/sentry/web/frontend/js_sdk_loader.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import absolute_import
+
+from sentry.relay import Config
+from sentry.models import ProjectKey
+from sentry.web.frontend.base import BaseView
+from sentry.web.helpers import render_to_response
+
+
+SDK_VERSION = '0.5.2'
+DEFAULT_SDK_URL = 'https://s3.amazonaws.com/getsentry-cdn/@sentry/browser/%s/bundle.min.js' % SDK_VERSION
+CACHE_CONTROL = 'public, max-age=30, s-maxage=60, stale-while-revalidate=315360000, stale-if-error=315360000'
+
+
+class JavaScriptSdkLoader(BaseView):
+    auth_required = False
+
+    def get(self, request, public_key, minified):
+        """Returns a js file that can be integrated into a website"""
+        key = ProjectKey.objects.get(
+            public_key=public_key
+        )
+
+        if minified is not None:
+            minified = True
+        else:
+            minified = False
+
+        config = Config(key.project)
+        context = {
+            'config': config.get_project_key_config(key),
+            'jsSdkUrl': DEFAULT_SDK_URL,
+            'minified': minified
```

Comment:
Wouldn't it be simpler to have two separate templates? I imagine potentially we'd want to minify the file during static file compilation in the future.